README.md

```diff
@@ -1,2 +1,81 @@
-# hanish
-A slackbot that makes small talk about the weather. 
+# Hanish
```

Comment:
When I teach programming, I usually say that the job of a programmer is about naming things!